### PR TITLE
update org paths & convert logos

### LINKED ISSUE #683: update paths/links according new org
## 🐛 Bug

With moving to the PyTorchLightning org we need to update builds, badges and other related links...

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?
Fixes #683. Updating paths according to the recent move to the organisation... #604

- update paths
- optimize imports
- convert logos

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

### CODE REVIEW

**Comment by Borda** [comment]
@williamFalcon how is the documentation deployment, what will be the new path?

**Comment by Borda** [comment]
@williamFalcon if seems that all bots are lost, can you add WIP bot...

**Comment by williamFalcon** [comment]
> how is the documentation deployment, what will be the new path

not sure. what do you suggest?

File: docs/source/_templates/theme_variables.jinja [hunk 1]
```diff
@@ -1,17 +1,17 @@
 {%- set external_urls = {
-  'github': 'https://github.com/williamFalcon/pytorch-lightning',
-  'github_issues': 'https://github.com/williamFalcon/pytorch-lightning/issues',
-  'contributing': 'https://github.com/williamFalcon/pytorch-lightning/blob/master/CONTRIBUTING.md',
-  'docs': 'https://williamfalcon.github.io/pytorch-lightning',
+  'github': 'https://github.com/PyTorchLightning/pytorch-lightning',
+  'github_issues': 'https://github.com/PyTorchLightning/pytorch-lightning/issues',
+  'contributing': 'https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CONTRIBUTING.md',
+  'docs': 'https://PyTorchLightning.github.io/pytorch-lightning',
```
Comment (Borda) [inline docs/source/_templates/theme_variables.jinja]: @williamFalcon this path/URL and the following linking specific section in new documentation...
Do you run it on readthecocs.org right?

File: README.md [hunk 2]
```diff
@@ -274,77 +274,77 @@ Lightning also adds a text column with all the hyperparameters for this experime
 
 ![tensorboard-support](docs/source/_static/images/tf_tags.png)
 
-## Lightning automates all of the following ([each is also configurable](https://williamfalcon.github.io/pytorch-lightning/Trainer/)):
+## Lightning automates all of the following ([each is also configurable](https://PyTorchLightning.github.io/pytorch-lightning/Trainer/)):
```
Comment (Borda) [inline README.md]: all the following links shall be revisited according newly generated docs...

**Comment by Borda** [comment]
> > how is the documentation deployment, what will be the new path
>
> not sure. what do you suggest?

I would say to keep the version of the last docs on your personal URL - https://PyTorchLightning.github.io/pytorch-lightning
and the new one plus all future on organisation one - https://williamfalcon.github.io/pytorch-lightning
just add some kind of warning that the old one is deprecated...

@PyTorchLightning/core-contributors any other suggestion?

File: README.md (hunk 2 above)
Comment (Borda) [reply README.md]: quite a number of links are referring just a single line in docs so I would skip them and if needed just list the topic on the page... @williamFalcon ?

Comment (tullie) [reply README.md]: What are you referring to exactly @Borda? Only one link per page? And not linking to each section fragment? 

If so, i'd agree with this. When joining this project I was pretty overwhelmed with how many links there were. I opened them all to start reading, then realized most pages are the same. 

Alternatively, if we were having links to all sections - I think the sections in the Readme should align with the sections in the docs. E.g. Training Loop should only have the Training Loop page links underneath.

Comment (Borda) [reply README.md]: that was my thoughts, have links only to major section and alternatively add examples/notebooks where the particular applications are explained...

**Comment by Borda** [comment]
the live docs is available here - https://torch-lightning.readthedocs.io/en/paths/
logo fix is here - https://github.com/PyTorchLightning/lightning_sphinx_theme/pull/1

@PyTorchLightning/core-contributors ^^

**Comment by Borda** [comment]
Did we test the TensorBoardLogger properly?
`AttributeError: 'SummaryWriter' object has no attribute 'get_data_path'`

**Comment by Borda** [comment]
waiting for https://github.com/PyTorchLightning/pytorch-lightning/pull/687 8-)

**Comment by williamFalcon** [comment]
@Borda the docs [here](https://torch-lightning.readthedocs.io/en/paths//pytorch_lightning.core.grads.html) are still unusable. let's hold off on adding to main README until they are usable and well organized.

**Comment by Borda** [comment]
@PyTorchLightning/core-contributors ^^
@williamFalcon or shall I do it as PR to yours #691 or just a cherry-pick?

**Comment by williamFalcon** [comment]
@Borda rebase master into this PR?

**Comment by Borda** [comment]
sure, just do not have internet for the weekend so I will finish it on Monday...
What is the final/temporal deployment address? https://pytorchlightning.github.io/pytorch-lightning

**Comment by williamFalcon** [comment]
final is here: https://pytorchlightning.github.io/pytorch-lightning

**Comment by Borda** [comment]
@williamFalcon updated...

**Comment by Borda** [comment]
as discussed on slack, lets use as docs path this one for now... https://pytorch-lightning.rtfd.io
 and  https://pytorchlightning.github.io/pytorch-lightning as a landing page...

**Comment by Borda** [comment]
@williamFalcon @neggert can we move this forward?

**Pull request merged**